atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1373,6 +1373,12 @@ void WebContents::SetSize(const SetSizeParams& params) {
     guest_delegate_->SetSize(params);
 }
 
+void WebContents::SetGuestSize(int width, int height) {
+  SetSizeParams params;
+  params.normal_size.reset(new gfx::Size(width, height));
+  SetSize(params);
```

Comment:
@kevinsawicki Done! I only documented the relevant part of `webContents.setSize`. I'm happy to document the other stuff (`min`, `max`, `enableAutoSize`) if you prefer.